product_listings_manager/tests/test_root.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+
+from product_listings_manager.app import app
+
+
+@pytest.yield_fixture
+def client():
+    client = app.test_client()
+    yield client
+
+
+class TestAbout(object):
```

Comment:
I think this class name should be TestRoot, there is no about link in this page

Reply:
Yes! I made a copy & paste mistake here. I've amended this to be `TestRoot` now.